cmd/operator/main.go

```diff
@@ -22,6 +24,7 @@ import (
 	metricsserver "sigs.k8s.io/controller-runtime/pkg/metrics/server"
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 
+	"github.com/go-logr/logr"
```

Comment:
Why is this package added?

Reply:
Same as `klog`, we need it so we can use `slog` on internal parts of the operator.